src/pages/commitqueue/codeChangesModule/CodeChangesModule.tsx

```diff
@@ -29,30 +29,32 @@ export const CodeChangeModule: React.VFC<{
 }> = ({ moduleCodeChange }) => {
   const { fileDiffs } = moduleCodeChange;
 
-  const modules = bucketByCommit(fileDiffs).map((commitDiffs) => {
-    const { description } = commitDiffs[0] ?? {};
-    return (
-      <CodeChangeModuleContainer key={`code_change_${description}`}>
-        {description && (
-          <CommitName data-cy="commit-name">{description}</CommitName>
-        )}
-        <Accordion
-          title={
-            <DropDownText>
-              <DropDownTextStyle>Total Code changes</DropDownTextStyle>
-              <CodeChangesBadge {...totalFileDiffs(commitDiffs)} />
-            </DropDownText>
-          }
-        >
-          <TableWrapper>
-            <CodeChangesTable fileDiffs={commitDiffs} showHeader={false} />
-          </TableWrapper>
-        </Accordion>
-      </CodeChangeModuleContainer>
-    );
-  });
-
-  return <>{modules}</>; // eslint-disable-line react/jsx-no-useless-fragment
+  return (
```

Comment:
We lost some alignment on preserve commits code changes (here's my original [patch](https://spruce.mongodb.com/version/621fdc0732f4176255f38986/changes?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC) to help with testing!)

<img width="362" alt="Screen Shot 2022-10-26 at 8 27 58 PM" src="https://user-images.githubusercontent.com/47064971/198163323-a379da2a-3830-4cf0-910f-4eaaebbd8f57.png">